src/util.rs

```diff
@@ -107,7 +107,7 @@ pub fn get_repo_description(root: &std::path::Path) -> Result<Option<(String, bo
 #[cfg(feature = "git2")]
 pub fn get_repo_head(
     root: &std::path::Path,
-) -> Result<Option<(Option<String>, String)>, git2::Error> {
+) -> Result<Option<(Option<String>, String, String)>, git2::Error> {
```

Comment:
The docs need a slight update: "head's reference-name, long-hash and short-hash"